lib/ansible/modules/windows/win_domain_group.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# this is a windows documentation stub, actual code lives in the .ps1
+# file of the same name
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: win_domain_group
+version_added: '2.4'
+short_description: creates, modifies or removes domain groups
+description:
+- Creates, modifies or removes groups in Active Directory.
+- For local groups, use the M(win_group) module instead.
+options:
+  name:
+    description:
+    - The name of the group to create, modify or remove.
+    - This value can be in the forms C(Distinguished Name), C(objectGUID),
+      C(objectSid) or C(sAMAccountName), see examples for more details.
+    required: yes
+  display_name:
+    description:
+    - The value to assign to the LDAP C(displayName) attribute.
+  username:
+    description:
+    - The username to use when interacting with AD.
+    - If this is not set then the user Ansible used to log in with will be
+      used instead.
+  password:
+    description:
+    - The password for C(username).
+  description:
+    description:
+    - The value to assigned to the LDAP C(description) attribute.
+  category:
+    description:
+    - The category of the group, this is the value to assign to the LDAP
+      C(groupType) attribute.
+    - If a new group is created then C(security) will be used.
```

Comment:
s/will be used/will be used by default if not otherwise specified/ - or could be handled like `scope` (no default, must be specified on new group)